### PR TITLE
APB-7856: Added welsh content for view contact details page

### PR DESCRIPTION
**Pull request by r-melvin**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -457,21 +457,25 @@ your.account.your.groups.off.p=Mae’r grwpiau mynediad wedi’u hanalluogi. Dim
...
-update-contact-details.summary-list.body.li4=a partner TBC
-update-contact-details.summary-list.body.li5=a sole trader TBC
-update-contact-details.summary-list.body.p2=We aim to update your contact details within 28 days. We’ll send you a letter when this is done. TBC
-update-contact-details.summary-list.email=Email address TBC
-update-contact-details.summary-list.phone=Telephone number TBC
-update-contact-details.summary-list.name=Business name TBC
-update-contact-details.summary-list.address=Business address TBC
-update-contact-details.summary-list.none=None TBC
+view-contact-details.title=Manylion cyswllt
+view-contact-details.heading=Manylion cyswllt
```
Comment (CarolineS-QA) [inline conf/messages.cy:461]: how come this is a duplicate key? why not just use one key for both heading & title

**Review by vaibhavidhami — DISMISSED** [review]
All Welsh content looks fine

File: conf/messages.cy (hunk 1 above)
Comment (r-melvin) [reply conf/messages.cy:461]: Oh that's a habit from previous teams, we sometimes had situations where they were different slightly. Happy to change it

Comment (CarolineS-QA) [reply conf/messages.cy:461]: Ahh, yea I can think of some cases where they'd need to be slightly different. But if it's the exact same I'd rather not have a duplicate. In general h1 & title should be the same (minus the service name - GOV.UK suffix) so using the same key where possible helps with that.

**Review by vaibhavidhami — APPROVED** [review]

**Pull request merged**